apps/dg/controllers/document_helper.js

```diff
@@ -206,6 +206,25 @@ DG.DocumentHelper = SC.Object.extend(
                     }
                   } else if (prop === 'rerandomize' && !!iValues[prop]) {
                     context.doRerandomizeAll();
+                  } else if (prop === 'sort') {
+                    var controllersMap = DG.currDocumentController().get('componentControllersMap');
+                    var tableAdapters = [];
+                    for (var key in controllersMap) {
+                      if (controllersMap[key].hasOwnProperty('caseTableAdapters')) {
+                        tableAdapters.push(controllersMap[key]);
+                        break;
+                      }
+                    }
+                    tableAdapters.forEach(function (tableController) {
+                      var tableContext = tableController.get('dataContext');
+                      var attribute = typeof(iValues[prop].attrId) === "string"
+                                        ? tableContext.getAttribute(iValues[prop].attrId).id
+                                        : iValues[prop].attrId;
```

Comment:
Do we need to `id => attribute => id` rather than just passing `iValues[prop].attrId` to `sortAttribute` below?

Reply:
This was to take into account that users can pass either the attribute name or the id.

Reply:
I don't see a `getAttribute` method in the code, but there is a `getAttributebyName()` method -- and then I'd suggest adding a comment about the fact that `attrId` can be a name or an id because otherwise one is tempted to assume that it's an id for some strange reason. 😉